doc/Data Plane Code Generation/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -1,201 +1,146 @@
 # Azure SDK Code Geneneration Quickstart Tutorial (Data Plane)
 The purpose of the Azure SDK is to provide a unified developer experience across Azure services. In this tutorial, we will walk through creating a new Azure SDK Generated Client library for a data plane Azure service.  The output library will have a standardized API based on the  [.NET Azure SDK Design Guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html).
```

Comment:
We should explain in the introduction what we mean by shippable and link to further information on the process and requirements for shipping if we choose to keep using that term.

Reply:
Moving to the new terminology, we no longer need this definition in the introduction.